src/widgets/WeatherWidget.css

```diff
@@ -17,7 +17,7 @@
 }
 
 .weatherImage {
-  width: 50%;
+  width: 95px;
```

Comment:
If you enlarge the widget, icon will use half of the widget. It's not useful. It's better to only increase space for text. What do you think?